plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaBuilderTest.kt

```diff
@@ -61,7 +61,7 @@ class JavaLambdaBuilderTest {
     }
 
     @Test
-    fun mavenRootPomHandlerBaseDirIsCorrect() = runTest {
+    fun mavenRootPomHandlerBaseDirIsCorrect(): Unit = runBlocking {
```

Comment:
tests should use runTest

Reply:
will revert it

Reply:
discussed offline, the issue is with `runTest` and the fix is in a later version of coroutines-test